cosmo_tester/framework/examples/__init__.py

```diff
@@ -134,15 +145,20 @@ def install(self):
     def clone_example(self):
         if not self._cloned_to:
             self._cloned_to = git_helper.clone(self.REPOSITORY_URL,
-                                               str(self.tmpdir),
+                                               os.path.join(
```

Comment:
Maybe let's calculate this path in a separate line, and then used here. Seems wasteful to add 5 lines where 2 would suffice.